### DISCUSSION TITLE
Subsystem Sphingolipid metabolism

### DISCUSSION THREAD

**Discussion by Flash1593** [SysBioChalmers/Human-GEM#438] [General]
The Sphingolipid metabolism subsystem in Human-GEM doesn't show the exchange reactions between cytoplasm and different organelles. Normally, in cells the sphingolipid molecules have a crosstalk between different organelles for their transport to membrane. Would it represent a true picture for modelling simulation of sphingolipid metabolism from Human-GEM?

**Comment by haowang-bioinfo** [comment]
The Sphingolipid metabolism subsystem in Human-GEM spans across cytoplasm and other [organelles](https://metabolicatlas.org/explore/Human-GEM/map-viewer/sphingolipid_metabolism?dim=2d&panel=0&sel=MAM02908r&search=&coords=-4028.63,-13601.74,0.13,0,0,500&dataTypes=None&dataSources=None&dataSets=None), including ER, nucleus, golgi apparatus, and lysosome. Genome-scale metabolic models, such as Human-GEM, categorize the crosstalking reactions between organelles as transport reactions, e.g. for metabolites [sphingosine](https://metabolicatlas.org/explore/Human-GEM/gem-browser/metabolite/MAM02929c), [9-O-acetylated-GD3](https://metabolicatlas.org/explore/Human-GEM/gem-browser/metabolite/MAM01244c).

As a scientific platform, Human-GEM relies on and evolves with experimental evidence for falsifiability.

**Comment by Flash1593** [comment]
So, does the Human-GEM have the information about the transport reactions involved in sphingolipid metabolism?

**Reply by haowang-bioinfo** [reply]
yes
